scanpipe/views.py

```diff
@@ -1086,6 +1089,8 @@ def perform_action(self, action, project_uuid, action_kwargs=None):
             raise Http404
 
     def get_success_message(self, action, count):
+        if action == "delete":
+            return f"{count} project{'s' if count != 1 else ''} {'is' if count == 1 else 'are'} being deleted in the background."
```

Comment:
I don't think the user needs to know about the internals of the deletion, let's keep the initial message.